### DISCUSSION TITLE
Force load of Datafeedr shortcode

### DISCUSSION THREAD

**Discussion by Misssthing** [datafeedr/wordpress-plugins#18] [Help]
To optimise website speed on Wordpress we're all using cache plugins and often adding an extra caching level via Cloudflare.
Often the Datafeedr comparison sets not load / refresh and show a "-1" for the visitor. 
Once you clear the (plugin) cache the issues is solved for a periode of time.
Any idea how I can 'force load' or prevent caching for the datafeedr comparison set shotcode? 
[dfrcs name="product name"]

**Comment by EricBusch** [comment]
Hi

Can you try setting your Display Method to PHP here WordPress Admin Area > Datafeedr API > Comparison Sets?

That should help.

If you have any other questions, please don't hesitate to ask.

Thanks,
Eric

**Reply by Misssthing** [reply]
Hi Eric, fast as ever. 
For this site the display method was indeed AJAX, I've changed this to PHP now. 
I do believe I've changed this before in trying to solve the issue, but since it was on AJAX now, ill try again.
ill monitor the issue and come back here to report in a while! thanks

**Reply by EricBusch** [reply]
OK, let us know! Thanks!

**Comment by Olovs** [comment]
Hey man,

I've had this problem for over a year as well. Changing to AJAX or PHP won't solve it.

I have tried changing server, theme, cache plugin, using no cache plugin and so on.

I've seen other people in Facebook groups having the same issue. It seems to be quite common. Unfortunately, it doesn't look like it's being worked on. I don't even want to know how much money I've missed out on because of this issue..

**Reply by EricBusch** [reply]
@Olovs I'm sorry to hear you are also having this issue!

Can you try adding this code to your [custom plugin](https://datafeedrapi.helpscoutdocs.com/article/32-creating-your-own-custom-plugin):

```php
add_filter( 'dfrcs_bots', function () {
	return [];
}, 10, 1 );
```

Hopefully this will resolve it for you! Be sure to clear your cache after adding that code.

Hope this helps!

Eric

**Reply by Olovs** [reply]
Hi Eric,

Thank you, I will try that and hope for the best.
